pkg/awxrunner/awx_action_runner.go

```diff
@@ -14,34 +14,73 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package main
+package awxrunner
 
 import (
 	"fmt"
 
 	"github.com/golang/glog"
+	"golang.org/x/sync/syncmap"
+	"k8s.io/apimachinery/pkg/util/wait"
 
 	"github.com/moolitayer/awx-client-go/awx"
 	"github.com/openshift/autoheal/pkg/alertmanager"
 	"github.com/openshift/autoheal/pkg/apis/autoheal"
+	"github.com/openshift/autoheal/pkg/config"
+	"github.com/openshift/autoheal/pkg/metrics"
 )
 
-func (h *Healer) runAWXJob(rule *autoheal.HealingRule, action *autoheal.AWXJobAction, alert *alertmanager.Alert) error {
-	var err error
+type AWXActionRunnerBuilder struct {
```

Comment:
As this has its own package now I think you can rename it to just `Builder`. Same for the runner, it can be just `Runner`.